pwiz_tools/Skyline/Model/Lib/NistLibSpec.cs

```diff
@@ -1789,6 +1823,91 @@ public override bool TryGetIrts(out LibraryRetentionTimes retentionTimes)
             return dictionary.Any();
         }
 
+        public override bool TryGetIonMobilityInfos(LibKey key, MsDataFileUri filePath, out IonMobilityAndCCS[] ionMobilities)
+        {
+            int i = FindEntry(key);
+            if (i != -1 && !IonMobilityAndCCS.IsNullOrEmpty(_libraryEntries[i].IonMobility))
+            {
+                ionMobilities = new[]{ _libraryEntries[i].IonMobility};
+                return true;
+            }
+
+            return base.TryGetIonMobilityInfos(key, filePath, out ionMobilities);
+        }
+
+        public override bool TryGetIonMobilityInfos(LibKey[] targetIons, MsDataFileUri filePath, out LibraryIonMobilityInfo ionMobilities)
+        {
+            ILookup<LibKey, IonMobilityAndCCS[]> ionMobilitiesLookup;

```

Comment:
I think this should just be:
`ILookup<LibKey, IonMobilityAndCCS>`
without the "[]" after "IonMobilityAndCCS".

Lookups are always a map between a single key and a list of values.
I don't think you want a list of list of values.

This code seems to have been copied from BiblioSpecLite.cs.